package.json

```diff
@@ -43,6 +43,7 @@
     "backbone.babysitter": "^0.1.0",
     "backbone.radio": "^0.9.0",
     "backbone": "1.0.0 - 1.1.2",
+    "backbone-metal": "^0.5.0",
```

Comment:
yea, i like that. we'll have a chance to do a new release at somepoint and make sure the next is legit